src/sonic-yang-models/yang-models/sonic-portchannel.yang

```diff
@@ -64,8 +64,8 @@ module sonic-portchannel {
 				}
 
 				leaf min_links {
-					type uint8 {
-						range 1..128;
+					type uint16 {
+						range 1..1024;
 					}
```

Comment:
Yang model subgroup meeting 4/8:
Do any tests need to be updated with new range limit (128->1024)? (cc: @praveen-li )

Reply:
Looks like a test case needs to be added for the range check.